spec/features/errors_spec.rb

```diff
@@ -6,15 +6,41 @@
 
     within '.govuk-main-wrapper' do
       expect(page).to have_css('.govuk-heading-xl', text: I18n.t('error.404_title'))
+      expect(page).to have_css('.govuk-body', text: 'If you typed the web address, check it is correct.')
+      expect(page).to have_css('.govuk-body',
+                               text: 'If you pasted the web address, check you copied the entire address.')
+      expect(page).to have_css('.govuk-body', text: 'You can also ')
+      expect(page).to have_css('.govuk-link', text: 'browse from the homepage')
```

Comment:
This could be removed, if the above comment is implemented

Reply:
This check also tests that the link has the `govuk-link` class.